surfinBH/_fit_evaluators/fit_7dq4Emri.py

```diff
@@ -0,0 +1,267 @@
+        
+        # All of these together (this fit returns vf = vf_err = None)
+        # This output is needed for compatibility with the other fits
+        mf, chif, vf, mf_err, chif_err, vf_err = 
+            fit.all(q, chiA, chiB, **kwargs)
+
+    The arguments for each of these call methods are as follows:
+    Arguments:
+        q:     Mass ratio (q = mA/mB >=1)
+
+        chiA:  Dimensionless spin vector of the heavier black hole at
+               reference epoch.
+
+        chiB:  Dimensionless spin vector of the lighter black hole at
+               reference epoch.
+
+                This follows the same convention as LAL, where the spin
+                components are defined as:
+                \chi_z = \chi \cdot \hat{L}, where L is the orbital angular
+                    momentum vector at the epoch.
+                \chi_x = \chi \cdot \hat{n}, where n = body2 -> body1 is the
+                    separation vector at the epoch. body1 is the heavier body.
+                \chi_y = \chi \cdot \hat{L \cross n}.
+                These spin components are frame-independent as they are defined
+                using vector inner products. This is equivalent to specifying
+                the spins in the coorbital frame at the reference epoch. 
+                For comparable-mass binaries the reference epoch is assumed to 
+                be at t=-100 M from the peak of the waveform, which corresponds 
+                to 2-4 GW cycles before the peak. In the EMRI limit the 
+                reference epoch is the ISCO (innermost stable circular orbit). 
+                In the intermediate regime the reference epoch is a transition 
+                between these two frames. See Sec. III in THE PAPER for 
+                additional details.
+
+     Inertial frame for returned values:
+
+        The returned chif are in the LAL inertial frame defined as follows:
+            The +ve z-axis is along the orbital angular momentum at the
+            reference epoch. The separation vector from the lighter BH to the
+            heavier BH at the reference epoch is along the +ve x-axis. The
+            y-axis completes the right-handed triad.
+
+            Note that the default reference epoch corresponds to t=-100M for 
+            comparable-mass binaries, the ISCO for EMRI binaries, and an 
+            intemediate epoch for binaries in between the two limits.
+            See LIGO DCC document T1800226 for the LAL frame diagram.
+    """
+
+    #-------------------------------------------------------------------------
+    def __init__(self, name):
+
+        # Param limits beyond which to raise a warning
+        soft_param_lims = {
+            'q': np.Inf,
+            'chiAmag': 1,
+            'chiBmag': 1,
+                }
+
+        # Param limits beyond which to raise an error
+        # In principle q has no limit, but an upper value is needed for testing
+        hard_param_lims = {
```

Comment:
The second line in the comment is obsolete now? Do the same thing here: error is never raised.

Reply:
Done